Add documentation about cli `add connection` and AWS connection URI (#28852)
mistercrunch/Airflow: docs/apache-airflow-providers-amazon/connections/aws.rst

    os.environ[env_key] = conn_uri
    print(conn.test_connection())

>   .. warning:: When using airflow CLI there might be a need to add ``@`` when:

Reviewer: ```suggestion
  .. warning:: When using the Airflow CLI, a ``@`` may need to be added when:
```

    - login
    - password,
    - host

>     - port

Reviewer: Why does only the password have a comma attached? Does this mean something I don’t get?

Author: 👍 good find

    os.environ[env_key] = conn_uri
    print(conn.test_connection())

  .. warning:: When using airflow CLI there might be a need to add ``@`` when:

    - login
    - password,
    - host
    - port

>     are not given see example below. This is a known airflow limitation.

Reviewer 2: Is there a comma or something missing here between `are not given` and  `see example below`?

>     ``airflow connections add aws_conn --conn-uri aws://@/?egion_name=eu-west-1``

Reviewer 3: ```suggestion
    ``airflow connections add aws_conn --conn-uri aws://@/?region_name=eu-west-1``
```
